docs/docs/commands/sync.md

```diff
@@ -32,6 +32,14 @@ notifications (security breaches, changes to service, etc).
 Upon success, you are also logged in :) Syncing should happen automatically from
 here!
 
+## Unregister
```

Comment:
```suggestion
## Delete
```

Reply:
What do we name the command, still `unregister` or `delete`?

Reply:
I'll follow up with a change, I've restructured the commands a bit!

Reply:
Okay cool, in that case I leave it as it is  😊